hnn_core/viz.py

```diff
@@ -569,13 +569,87 @@ def plot_cell_morphology(cell, ax, show=True):
     return ax
 
 
+def plot_connectivity_weights(net, conn_idx, ax=None, show=True):
```

Comment:
should we still have the `plot_connectivity_matrix` function? Shouldn't this be absorbed into this one?

Reply:
Yeah I was realizing it is pretty superfluous while writing this. I'll add a flag that skips the weight computation and just plots black/white squares.

Then this can be renamed `plot_connectivity_matrix`.

Reply:
I think you might want to ignore this comment now since [I understand](https://github.com/jonescompneurolab/hnn-core/pull/339#issuecomment-848837244) the situation better now.

Reply:
this function is still relevant after getting the `cell-specific` plotting function?

Reply:
Honestly I am still leaning towards my previous comment in this thread. While the weight visualization here is obviously not as intuitive as cell-specific plotting, it seems weird not to include the functionality. 

I'm planning to remove the "black and white" matrix function, and rename this function to `plot_connectivity_matrix`. Then I can set a `show_weights=False` flag for the default behavior.